entrypoint.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+
+if [ "$SPIDER_TO_RUN" == 'who_iris' ]
+then
+    scrapy crawl who_iris
+elif [ "$SPIDER_TO_RUN" == 'nice' ]
+then
+    scrapy crawl nice
+else
+    echo "Either you did not specifiy a spider, or the spider you want to run does not exist."
```

Comment:
sp: “specifiy”

`set -o nonuset` should protect you from the `$SPIDER_TO_RUN` variable being empty.

You should log the name of the spider the user provided to make debugging easier.

Reply:
I'm already re-writing this for another feature 👍